config/environments/default.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from fsd_utils import configclass
+
+
+@configclass
+class DefaultConfig:
+
+    if "FLASK_ENV" in os.environ:
+        FLASK_ENV = os.getenv("FLASK_ENV")
+    else:
+        raise KeyError("FLASK_ENV does not exist in environ")
```

Comment:
🤔 Isn't FLASK_ENV set outside of config? Its what decides on which config is loaded.

Reply:
Yes - but don't we also need to load it into the Config object?

Reply:
I think Flask loads it at start up